contracts/EmployeeTokenOwnershipPlan2020.sol

```
@@ -0,0 +1,212 @@
+            0xaBad5427278F99c9b9393Cc46FDb0Cb4CB6C33f5,
+            0x87adb1BEa935649E607f615F41ae8f4cA96566fa,
+            0x6D0228303D0608CACc8a262deA95932DCAc12c8D,
+            0x49c268e3F2119fCf71f70dF987432689dd4145Ad,
+            0x24C08921717bf5C0029e2b8013B70f1D203cCDac
+        ];
+
+        uint80[45] memory _amounts = [
+            308310 ether,
+            453078 ether,
+            485991 ether,
+            538180 ether,
+            573806 ether,
+            482910 ether,
+            517196 ether,
+            330598 ether,
+            519363 ether,
+            530065 ether,
+            470891 ether,
+            667795 ether,
+            730172 ether,
+            750079 ether,
+            500053 ether,
+            145641 ether,
+            775175 ether,
+            180661 ether,
+            340060 ether,
+            398740 ether,
+            120010 ether,
+            692576 ether,
+            384004 ether,
+            475260 ether,
+            187520 ether,
+            150834 ether,
+            31254 ether,
+            398740 ether,
+            435961 ether,
+            500972 ether,
+            549381 ether,
+            561055 ether,
+            221724 ether,
+            375040 ether,
+            425292 ether,
+            433972 ether,
+            459366 ether,
+            501058 ether,
+            539577 ether,
+            750079 ether,
+            824272 ether,
+            750083 ether,
+            797479 ether,
+            1076356 ether,
+            162000 ether
+        ];
+
+        uint _totalReward = 21502629 ether;
+        vestStart = block.timestamp;
+
+        for (uint i = 0; i < _members.length; i++) {
+            Record memory record = Record(block.timestamp, _amounts[i], 0);
+            records[_members[i]] = record;
+            totalReward = totalReward.add(_amounts[i]);
+        }
+        require(_totalReward == totalReward, "VALUE_MISMATCH");
```

If addresses aren't unique this will not work correctly and the `totalReward` check does not catch the error.

---

All addresses are assumed to be unique, I think.

---

We should check the new address is not yet in the record. Otherwise, the contract should throw an error.